package/src/test.rs

```diff
@@ -636,3 +645,81 @@ fn test_self_downgrade(scie_pants_scie: &Path) {
     )
     .unwrap();
 }
+
+fn test_caching_issue_129(scie_pants_scie: &Path) {
+    integration_test!(
+        "Verifying the build root does not influence caching ({issue})",
+        issue = issue_link(129)
+    );
+    let tmpdir = create_tempdir().unwrap();
+
+    let scie_base = tmpdir.path().join("nce");
+
+    let pants_toml = r#"
+    [GLOBAL]
+    pants_version = "2.15.0"
+    [anonymous-telemetry]
+    enabled = false
+    "#;
+
+    let one = tmpdir.path().join("one");
+    ensure_directory(&one, false).unwrap();
+    write_file(&one.join("pants.toml"), false, pants_toml).unwrap();
+    execute(
+        Command::new(scie_pants_scie)
+            .arg("-V")
+            .env("SCIE_BASE", &scie_base)
+            .current_dir(&one),
+    )
+    .unwrap();
+
+    let two = tmpdir.path().join("two");
+    ensure_directory(&two, false).unwrap();
+    write_file(&two.join("pants.toml"), false, pants_toml).unwrap();
+    execute(
+        Command::new(scie_pants_scie)
+            .arg("-V")
+            .env("SCIE_BASE", &scie_base)
+            .current_dir(&two),
+    )
+    .unwrap();
+
+    #[derive(Debug, Eq, PartialEq)]
```

Comment:
This is a pretty indirect and fragile test - it knows alot of details about the implementation. I don't know of a good way to shut down the network for the span of this test though; so this does the trick.